js/process.ts

```diff
@@ -59,6 +61,21 @@ export class Process {
     return await runStatus(this.rid);
   }
 
+  /** Buffer the stdout and return it as Uint8Array after EOF.
+   * You must have set stdout to "piped" in when creating the process.
+   * This calls close() on stdout after its done.
```

Comment:
The close is in a try-finally block, so I believe that would actually work.  But yes, it's a debatable behavior....Go's cmd.Output does not close the pipe, https://golang.org/pkg/os/exec/#Cmd.Output but in Go you specify the pipe explicitly where as here the system creates the pipe for you...

Reply:
We definitely need to make a second pass at the subprocess API.

Reply:
> The close is in a try-finally block, so I believe that would actually work

That's not what I mean.
The problem is that the user needs to call output() at all, otherwise the handle leaks.
In many cases there'd be no point in calling output() if the process failed.